src/lib/containers/RssFeedCards.tsx

```diff
@@ -85,9 +85,9 @@ export default class RssFeedCards extends React.Component<RssFeedCardsProps, Rss
     return (
       <>        
         {this.props.mailChimpUrl && (
-          <div className="RssFeedSubscribe text-center">
+          <div className="FeedSubscribe text-center">
```

Comment:
Change all "RssFeed*" css class names to just "Feed*" and use them in both components.